projects/fsastorefrontlib/src/cms-components/my-account/policy/policies/policies.component.ts

```diff
@@ -40,6 +40,17 @@ export class PoliciesComponent implements OnInit, OnDestroy {
   ngOnInit() {
     this.policyService.loadPolicies();
     this.policies$ = this.policyService.getPolicies();
+    this.subscription.add(
+      this.policies$
+        .pipe(
+          filter((policies: any) => !!policies.insurancePolicies),
+          take(1),
+          tap((policies: any) =>
+            this.policyService.setPolicies(policies.insurancePolicies)
```

Comment:
When policies arrived I set `Subject` to supply them in the policies-chart component instead of two API calls.

Reply:
Are you sure that it will be 2 API calls if you try to use getPolicies() method for both cases? I think we had same situation on some different pages (getUserRequest I think) and we implemented facade logic so in case there are multiple execution of get method only one request go towards back-end

Reply:
You are right, there is only one API call, I don't need Subject.